src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -1174,7 +1183,7 @@ private static void ParseAndResolveReferencePaths(string switchName, string swit
             }
         }
 
-        private static string GetWin32Setting(string arg, string value, List<Diagnostic> diagnostics)
+        private string GetWin32Setting(string arg, string value, List<Diagnostic> diagnostics)
```

Comment:
No.  I think that is a hold over from an earlier change where it did.  I will move it back.